packages/neutrino/bin/neutrino.js

```diff
@@ -1,130 +1,9 @@
-#!/usr/bin/env node
+#! /usr/bin/env node
 
-// TODO: Remove this once babel-loader updates
-// https://github.com/babel/babel-loader/pull/391
-process.noDeprecation = true;
+console.error(`
+The "neutrino" command has been removed starting with v9.
+Please see the migration guide at https://neutrinojs.org/migration-guide
+for details on upgrading your installation.
+`);
 
-const yargs = require('yargs');
-const {
-  cond, equals, map, omit, T
-} = require('ramda');
-const build = require('./build');
-const inspect = require('./inspect');
-const start = require('./start');
-const test = require('./test');
-const execute = require('./execute');
-const { exists, req } = require('../src/utils');
-
-const cwd = process.cwd();
-const cli = yargs
-  .usage('Usage: $0 <command> [options]')
-  .help(false)
-  .option('help', {
-    description: 'Show help',
-    boolean: true,
-    default: false,
-    global: true
-  })
-  .option('inspect', {
-    description: 'Output a string representation of the configuration used by webpack and exit',
-    boolean: true,
-    default: false,
-    global: true
-  })
-  .option('use', {
-    description: 'A list of Neutrino middleware used to configure the build',
-    array: true,
-    default: [],
-    global: true
-  })
-  .option('options', {
-    description: 'Set Neutrino options, config, and environment variables, e.g. --options.env.NODE_ENV production',
-    default: {},
-    global: true
-  })
-  .option('quiet', {
-    description: 'Disable console output of CLI commands',
-    boolean: true,
-    default: false,
-    global: true
-  })
-  .option('debug', {
-    description: 'Run in debug mode',
-    boolean: true,
-    default: false,
```

Comment:
With this PR `neutrino.options.debug` now defaults to `undefined` rather than `false`, which means `clean-webpack-plugin` now gives verbose output (its default) when it didn't before. I think we need to make it default to `false` again.

On that note, `process.env.NODE_ENV` is also `undefined` when it wasn't before. I think we need to set that explicitly again too, per:
https://github.com/mozilla-neutrino/neutrino-dev/pull/852#pullrequestreview-119629134